LinksAwakeningClient.py

```diff
@@ -570,9 +579,11 @@ async def deathlink():
 async def main():
     parser = get_base_parser(description="Link's Awakening Client.")
     parser.add_argument("--url", help="Archipelago connection url")
+    parser.add_argument("--nomagpie", default=False, action='store_true', help="Disable magpie bridge")

```

Comment:
`--no-magpie` please

Reply:
Even better
 `   parser.add_argument("--no-magpie", dest='magpie", default=False, action='store_false`, help="Disable magpie bridge")`
then pass `args.magpie` to the context.